app/api/speaker_image_sizes.py

```diff
@@ -12,7 +13,17 @@ class SpeakerImageSizeDetail(ResourceDetail):
     """
     @classmethod
     def before_get(self, args, kwargs):
-        kwargs['id'] = 2
+        speaker_image_size, is_created = db_helper.get_or_create(ImageSizes, type='speaker-image')
+        if is_created:
+            speaker_image_size.icon_size_width_height = 35
+            speaker_image_size.icon_size_quality = 80
+            speaker_image_size.small_size_width_height = 50
+            speaker_image_size.small_size_quality = 80,
+            speaker_image_size.thumbnail_size_width_height = 500
+            speaker_image_size.thumbnail_quality = 80
+            db.session.commit()
```

Comment:
And all this logic is repeated. This should be in a single place throughout project and reused in the tasks as well

Reply:
Can you state an example, have we done for any other models?